rocksdb_replicator/rocksdb_replicator.h

```diff
@@ -52,6 +52,9 @@ namespace wangle {
 
 namespace replicator {
 
+const uint32_t kMinReplTimeoutMs = 1;
+const uint32_t kNumReplTimeoutsBeforeDegradation = 100;
```

Comment:
Should we make this configurable using a gflag? For high traffic clusters, I'm wondering if 100 is too high as a threshold?